cartographer_ros/cartographer_ros/assets_writer_main.cc

```diff
@@ -70,6 +70,7 @@ namespace carto = ::cartographer;
 carto::sensor::PointCloudWithIntensities ToPointCloudWithIntensities(
     const sensor_msgs::PointCloud2::ConstPtr& message) {
   pcl::PointCloud<pcl::PointXYZ> pcl_point_cloud;
+  fromROSMsg(*message, pcl_point_cloud);
   pcl::fromROSMsg(*message, pcl_point_cloud);
```

Comment:
Yeah, silliness.  This was part of my original porting effort, but it turns out that for "live" cartographer, you don't need this file, and it is never compiled.  In point of fact, we can't port this file to ROS2 until we have a story around rosbags.  So I'll just drop this change.